src/main/java/com/jvm_bloggers/core/rss/fetchers/WgetFetcher.java

```diff
@@ -0,0 +1,40 @@
+package com.jvm_bloggers.core.rss.fetchers;
+
+import com.rometools.rome.feed.synd.SyndFeed;
+import com.rometools.rome.io.SyndFeedInput;
+
+import javaslang.control.Try;
+
+import lombok.extern.slf4j.Slf4j;
+
+import org.springframework.stereotype.Component;
+
+import java.io.File;
+
+/**
+ * Execute wget unix command, save as rss a file in tmp directory
+ * and create SyndFeed from it
+ */
+@Slf4j
+@Component
```

Comment:
This is the default value, but You are right, it would be better to specify it explicitly.